packages/@ember/-internals/glimmer/tests/integration/components/angle-bracket-invocation-test.js

```diff
@@ -836,6 +836,28 @@ if (EMBER_GLIMMER_ANGLE_BRACKET_INVOCATION) {
         });
       }
 
+      '@test `...attributes` are merged regardless of the position where they are defined'() {
+        this.registerComponent('foo-bar', {
+          ComponentClass: Component.extend({ tagName: '' }),
+          template:
+            '<div class="inner" ...attributes>hello</div><div ...attributes class="inner">hello</div>',
+        });
+
+        this.render('<FooBar class="outer" />');
+
+        this.assertElement(this.firstChild, {
+          tagName: 'div',
+          attrs: { class: classes('inner outer') },
+          content: 'hello',
+        });
+
+        this.assertElement(this.nthChild(1), {
+          tagName: 'div',
+          attrs: { class: classes('inner outer') },
```

Comment:
after [trying this in glimmer-vm](https://github.com/GavinJoyce/glimmer-vm/pull/1), it appears that this should be `outer inner`

Reply:
^ updated